lib/openAi.ts

```diff
@@ -22,3 +29,18 @@ export const chatWithOpenAi = async (content: string) => {
   });
   return chatCompletion.choices[0].message.content!;
 };
+
+export const getParsedTicketAnalysis = async (
+  content: string,
+  prompt: string,
+) => {
```

Comment:
add parameter to pass outputSchema and then pass TicketAnalysisSchema from api route

Reply:
Done